### PR TITLE
Introduce PSR-4 autoloading

### PR DESCRIPTION
**Pull request by lukareihl**

### CODE REVIEW

File: examples/CustomFlowElement.php [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+ * the FlowElement takes the results of a client side form collecting
+ * date of birth, setting this as evidence on a FlowData object to calculate
+ * a person's starsign. The FlowElement also serves additional JavaScript
+ * which gets a user's geolocation and saves the latitude as a cookie.
+ * This latitude is also then passed in to the FlowData to calculate if
+ * a person is in the northern or southern hemispheres.
+
+ */
 
 include(__DIR__ . "/../vendor/autoload.php");
```
Comment (justadreamer) [inline examples/CustomFlowElement.php]: do we still need this include here?

**Review by justadreamer — CHANGES_REQUESTED** [review]
also if possible could we remove the newline character changes before committing in the file header comments - so it does not pollute the diff?

File: examples/CustomFlowElement.php (hunk 1 above)
Comment (lukareihl) [reply examples/CustomFlowElement.php]: Good catch, removed.

File: examples/Pipeline.php [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
-* This example demonstrates how various FlowElements can be combined in a Pipeline and processed
-*
-*/
+ * @example Pipeline.php
+ *
+ * This example demonstrates how various FlowElements can be combined in a Pipeline and processed
+ *
+ */
 
 require(__DIR__ . "/../vendor/autoload.php");
```
Comment (justadreamer) [inline examples/Pipeline.php]: perhaps this one can also be removed?

**Review by justadreamer — APPROVED** [review]

**Pull request merged**